exporter/datadogexporter/factory.go

```diff
@@ -79,8 +79,6 @@ func enableZorkianMetricExport() error {
 	return featuregate.GlobalRegistry().Set(metricExportNativeClientFeatureGate.ID(), false)
 }
 
-const metadataReporterPeriod = 30 * time.Minute
```

Comment:
I asked the same question about this last year.. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24290#discussion_r1265612835

@mx-psi is this a requirement from backend etc.?

Reply:
Yeah, in my testing, I set the period to 2 minutes and observed no issues with the Datadog backend. By default, the period is set to 30 minutes, but users can adjust it to a shorter interval if they need more frequent updates.